Failed to build on macOS

```
[ERROR] Failed to execute goal sh.tak.appbundler:appbundle-maven-plugin:1.2.0:bundle (default) on project semux: Execution default of goal sh.tak.appbundler:appbundle-maven-plugin:1.2.0:bundle failed: An API incompatibility was encountered while executing sh.tak.appbundler:appbundle-maven-plugin:1.2.0:bundle: java.lang.ExceptionInInitializerError: null
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>sh.tak.appbundler:appbundle-maven-plugin:1.2.0
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/Users/dev/.m2/repository/sh/tak/appbundler/appbundle-maven-plugin/1.2.0/appbundle-maven-plugin-1.2.0.jar
[ERROR] urls[1] = file:/Users/dev/.m2/repository/org/codehaus/plexus/plexus-velocity/1.1.8/plexus-velocity-1.1.8.jar
[ERROR] urls[2] = file:/Users/dev/.m2/repository/junit/junit/3.8.1/junit-3.8.1.jar
[ERROR] urls[3] = file:/Users/dev/.m2/repository/commons-collections/commons-collections/3.1/commons-collections-3.1.jar
[ERROR] urls[4] = file:/Users/dev/.m2/repository/velocity/velocity/1.5/velocity-1.5.jar
[ERROR] urls[5] = file:/Users/dev/.m2/repository/commons-lang/commons-lang/2.1/commons-lang-2.1.jar
[ERROR] urls[6] = file:/Users/dev/.m2/repository/oro/oro/2.0.8/oro-2.0.8.jar
[ERROR] urls[7] = file:/Users/dev/.m2/repository/org/codehaus/plexus/plexus-archiver/3.0.3/plexus-archiver-3.0.3.jar
[ERROR] urls[8] = file:/Users/dev/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.22/plexus-utils-3.0.22.jar
[ERROR] urls[9] = file:/Users/dev/.m2/repository/org/codehaus/plexus/plexus-io/2.7/plexus-io-2.7.jar
[ERROR] urls[10] = file:/Users/dev/.m2/repository/commons-io/commons-io/2.2/commons-io-2.2.jar
[ERROR] urls[11] = file:/Users/dev/.m2/repository/org/apache/commons/commons-compress/1.10/commons-compress-1.10.jar
[ERROR] urls[12] = file:/Users/dev/.m2/repository/org/iq80/snappy/snappy/0.4/snappy-0.4.jar
[ERROR] urls[13] = file:/Users/dev/.m2/repository/javax/enterprise/cdi-api/1.0/cdi-api-1.0.jar
[ERROR] urls[14] = file:/Users/dev/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.inject/0.3.2/org.eclipse.sisu.inject-0.3.2.jar
[ERROR] urls[15] = file:/Users/dev/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar
[ERROR] urls[16] = file:/Users/dev/.m2/repository/org/apache/maven/maven-builder-support/3.3.9/maven-builder-support-3.3.9.jar
[ERROR] urls[17] = file:/Users/dev/.m2/repository/com/google/guava/guava/18.0/guava-18.0.jar
[ERROR] urls[18] = file:/Users/dev/.m2/repository/org/eclipse/aether/aether-util/1.0.2.v20150114/aether-util-1.0.2.v20150114.jar
[ERROR] urls[19] = file:/Users/dev/.m2/repository/com/google/inject/guice/4.0/guice-4.0-no_aop.jar
[ERROR] urls[20] = file:/Users/dev/.m2/repository/aopalliance/aopalliance/1.0/aopalliance-1.0.jar
[ERROR] urls[21] = file:/Users/dev/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.21/plexus-interpolation-1.21.jar
[ERROR] urls[22] = file:/Users/dev/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.6/plexus-component-annotations-1.6.jar
[ERROR] urls[23] = file:/Users/dev/.m2/repository/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar
[ERROR] urls[24] = file:/Users/dev/.m2/repository/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
```

--- a/pom.xml
+++ b/pom.xml
@@ -324,7 +324,7 @@
                             <headerType>gui</headerType>
                             <jar>${project.build.directory}/${project.artifactId}-${project.version}-shaded.jar</jar>
                             <outfile>${project.build.directory}/semux.exe</outfile>
-                            <downloadUrl>http://java.com/download</downloadUrl>
+                            <downloadUrl>http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html</downloadUrl>
                             <classPath>
                                 <mainClass>org.semux.gui.SemuxGui</mainClass>
                                 <preCp>anything</preCp>
@@ -357,38 +357,6 @@
                                 <mutexName>semux</mutexName>
                             </singleInstance>
                         </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-
-            <!-- build macos executable -->
-            <plugin>
-                <groupId>sh.tak.appbundler</groupId>
-                <artifactId>appbundle-maven-plugin</artifactId>
-                <version>1.2.0</version>
-                <configuration>
-                    <mainClass>org.semux.wrapper.Wrapper</mainClass>
-                    <workingDirectory>$APP_ROOT/..</workingDirectory>
-                    <!-- <generateDiskImageFile>true</generateDiskImageFile> -->
-                    <jvmVersion>1.8</jvmVersion>
-                    <version>${project.version}-${git.commit.id.abbrev}</version>
-                    <iconFile>logo.icns</iconFile>
-                    <additionalResources>
-                        <additionalResource>
-                            <includes>
-                                <include>config/**</include>
-                                <include>LICENSE*</include>
-                            </includes>
-                            <directory>${project.basedir}</directory>
-                        </additionalResource>
-                    </additionalResources>
-                </configuration>
-                <executions>
-                    <execution>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>bundle</goal>
-                        </goals>
                     </execution>
                 </executions>
             </plugin>
@@ -681,7 +649,6 @@
                                         <urn>org.xerial.snappy:snappy-java:1.1.7.1:jar:null:compile:d5190b41f3de61e3b83d692322d58630252bc8c3</urn>
                                         <urn>org.yaml:snakeyaml:1.18:jar:null:compile:e4a441249ade301985cb8d009d4e4a72b85bf68e</urn>
                                         <urn>pl.project13.maven:git-commit-id-plugin:2.2.4:maven-plugin:null:runtime:43a0dc1d02821c51847d587ecc4a6943086b884c</urn>
-                                        <urn>sh.tak.appbundler:appbundle-maven-plugin:1.2.0:maven-plugin:null:runtime:75a9e8c2adc7203b9b443b4547570da25c619f0d</urn>
                                     </urns>
                                 </digestRule>
                             </rules>

